### ISSUE TITLE
Crashing with movit on 18.12.1 

### ISSUE THREAD

**Issue by jman005** [flathub/org.kde.kdenlive#27]
While using Kdenlive 18.12.1 from flatpak with movit GPU acceleration enabled, trying to click anywhere on the timeline or playback window results in a crash. Disabling movit resolves this. The console output on-crash follows: 
```
qt.qpa.xcb: QXcbConnection: XCB error: 8 (BadMatch), sequence: 10528, resource id: 60817733, major code: 130 (Unknown), minor code: 3
qt.qpa.xcb: QXcbConnection: XCB error: 8 (BadMatch), sequence: 10559, resource id: 60817733, major code: 130 (Unknown), minor code: 3
qt.qpa.xcb: QXcbConnection: XCB error: 8 (BadMatch), sequence: 10562, resource id: 60817733, major code: 130 (Unknown), minor code: 3
qt.qpa.xcb: QXcbConnection: XCB error: 8 (BadMatch), sequence: 11829, resource id: 60817733, major code: 130 (Unknown), minor code: 3
qt.qpa.xcb: QXcbConnection: XCB error: 8 (BadMatch), sequence: 11860, resource id: 60817733, major code: 130 (Unknown), minor code: 3
qt.qpa.xcb: QXcbConnection: XCB error: 8 (BadMatch), sequence: 11863, resource id: 60817733, major code: 130 (Unknown), minor code: 3
kdenlive: effect_chain.cpp:1662: void movit::EffectChain::render_to_fbo(GLuint, unsigned int, unsigned int): Assertion `status == GL_FRAMEBUFFER_COMPLETE' failed.
```

Movit on the appimage doesn't function at all (it's always greyed out), so I'm unable to test it with other releases. I'm using the RTX 2070 GPU (Founders' Edition) and an Intel Core i7 4770k on Ubuntu 18.04.

**Comment by eszlari** [comment]
I think this is an upstream (MLT/Movit) problem. MLT still only supports an old Movit version (1.2.0), while upstream is 1.6.2.

MLT's developer has even disabled Movit for his own editor:
https://github.com/mltframework/shotcut/commit/de101a1c5dc3efc4016f69dc2cbecdf73e75ae76

Maybe you should file a kdenlive bug, to do the same.

**Comment by tannisroot** [comment]
This issue seems specific to the Flatpak version. I've just tried Appimage and I can't reproduce the crash with it.

**Comment by eszlari** [comment]
I have updated the beta branch:
https://flathub.org/builds/#/builders/32/builds/1703

(There this crash is fixed)

**Comment by PatrickMalolepszy** [comment]
Glad to hear this fix is getting pushed out.

Is there a way to install the beta build in the meantime?

**Comment by tannisroot** [comment]
@PatrickMalolepszy 
```
flatpak remote-add flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak install flathub-beta org.kde.kdenlive
```

**Comment by eszlari** [comment]
This is now fixed with 19.04.
